refactor: use native uuid generator.

ghimirebibek opened the pull request.

rafaelsideguide reviewed (changes requested):
Is it possible to remove the uuid module entirely?

ghimirebibek commented:
> Is it possible to remove the uuid module entirely?

I've added a regex validation in place of uuid package's validate method. I've also removed uuid from the dependencies.
